pkgs/development/libraries/symengine/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, gmp
+, flint
+}:
+
+stdenv.mkDerivation rec {
+  name = "symengine-${version}";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "symengine";
+    repo = "symengine";
+    rev = "v${version}";
+    sha256 = "1p7hk163rgn1zzvjlq4vskblry3s2rg5bc7xlr08wfqckfr47bqc";
+  };
+
+  buildInputs = [ cmake gmp flint ];
```

Comment:
`cmake` & `flint` - `nativeBuildInputs` surely? (not sure how `gmp` is used here, but probably it too)